mslib/mscolab/conf.py

```diff
@@ -117,8 +117,8 @@ class default_mscolab_settings:
     # enable login by identity provider
     USE_SAML2 = False
 
-    # SSL certificates verification during SSO.
-    VERIFY_SSL_CERT = True
+    # Enable SSL certificate verification during SSO between MSColab and IdP
+    ENABLE_SSO_SSL_CERT_VERIFICATION = True
 
```

Comment:
Have a look if the line is missing there
https://github.com/Open-MSS/MSS/blob/develop/docs/samples/config/mscolab/mscolab_settings.py.sample

Reply:
Added. Thank you